ReactiveCocoa/CocoaAction.swift

```diff
@@ -19,7 +27,19 @@ public final class CocoaAction: NSObject {
 	/// This property will only change on the main thread, and will generate a
 	/// KVO notification for every change.
 	public private(set) var isExecuting: Bool = false
-	
+
+	/// Creates a producer for the `enabled` state of a CocoaAction.
+	public var isEnabledProducer: SignalProducer<Bool, NoError> {
+		return values(forKeyPath: #keyPath(CocoaAction.isEnabled))
+			.map { $0 as! Bool }
+	}
+
+	/// Creates a producer for the `executing` state of a CocoaAction.
+	public var isExecutingProducer: SignalProducer<Bool, NoError> {
+		return values(forKeyPath: #keyPath(CocoaAction.isExecuting))
+			.map { $0 as! Bool }
+	}
```

Comment:
Done in https://github.com/ReactiveCocoa/ReactiveCocoa/pull/3210/commits/6952c5310449f27a8ddd3ac199bbf0c2872dcc41.